components/user_settings/display/index.ts

```diff
@@ -54,7 +56,13 @@ function mapStateToProps(state) {
     };
 }
 
-function mapDispatchToProps(dispatch) {
+type Actions = {
```

Comment:
Looks like this isn't being used, so can we remove it?

Reply:
yes, it can be removed and I have removed it.